app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/ruling/ruling_explanation.scala.html

```diff
@@ -0,0 +1,55 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.tariffclassificationfrontend.controllers.routes.{PdfDownloadController, RulingController}
+@import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
+@import uk.gov.hmrc.tariffclassificationfrontend.models.{BTIApplication, Case, CaseStatus, Decision, StoredAttachment}
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.{attachments_list, tab_heading}
+@import uk.gov.hmrc.tariffclassificationfrontend.forms.DecisionFormData
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.ruling.{ruling_certificate_link, complete_ruling_section}
+@import uk.gov.hmrc.tariffclassificationfrontend.models.request.AuthenticatedRequest
+
+@(c: Case)(implicit request: AuthenticatedRequest[_], messages: Messages, appConfig: AppConfig)
+
+@if(c.decision.isDefined) {
+    @defining(c.decision.get) { d: Decision =>
+        <h3 class="heading-medium">@messages("case.decision.holder.information")</h3>
+        <div class="grid-row" id="information-for-holder">
+            <dt class="column-one-third">
+                @messages("case.decision.explanation")
+            </dt>
+            <dd class="column-two-thirds">
+            @if(d.explanation.exists(_.trim.nonEmpty)) {
```

Comment:
Do we need the `_trim.nonEmpty` if we mandate that is always defined+nonempty with form validation?

Reply:
I think it needs to be there when looking at a case that may have existed before the change as it would have effectively bypassed that validation

Reply:
If the case existed before this change it will have explanation set to `None` :)